build/ci/.azure-devops-unit-tests.yml

```diff
@@ -111,5 +111,3 @@ jobs:
       vsTestVersion: toolsInstaller
       testRunTitle: $(Agent.JobName)
       testSelector: testAssemblies
-      batchingBasedOnAgentsOption: customBatchSize
-      customBatchSizeValue: 200 # test count / 10 (https://developercommunity.visualstudio.com/content/problem/891803/vstestconsoleadapter-fails-with-outofmemory.html?childToView=896206#comment-896206)
```

Comment:
@jeromelaban Batching was the root cause of masking some test failures. As you see in this PR, it's properly failing the job. Aside, batching was slowing down the job, not really making it faster. Aside, things are even faster with the switch to MTP that I'm completing in another PR.

This is just for your information, no need to merge this PR as the other PR does everything needed ;)